client/tests/integration_test.rs

```diff
@@ -119,34 +119,24 @@ impl MockServerHandle {
         }
     }
 
-    fn get_balance(&self, state_root_hash: &str, purse_uref: &str) -> Result<(), ErrWrapper> {
-        casper_client::get_balance("1", &self.url(), 0, state_root_hash, purse_uref)
-            .map(|_| ())
-            .map_err(ErrWrapper)
+    fn get_balance(&self, state_root_hash: &str, purse_uref: &str) -> Result<(), Error> {
+        casper_client::get_balance("1", &self.url(), 0, state_root_hash, purse_uref).map(|_| ())
```

Comment:
I think you can get rid of this `.map` here, cause you don't need to create an `ErrWrapper`

Reply:
I left those in there because the functions returned `Result<(), Error>`, I could go back and change everything to `Result<T, Error>` depending on what client function is being called if that's better.